pathology/multiple_instance_learning/README.md

```diff
@@ -0,0 +1,119 @@
+# Multiple Instance Learning (MIL) Examples
+
+This tutorial contains a baseline method of Multiple Instance Learning (MIL) classification from Whole Slide Images (WSI).
+The dataset is from  [Prostate cANcer graDe Assessment (PANDA) Challenge - 2020](https://www.kaggle.com/c/prostate-cancer-grade-assessment/) for cancer grade classification from prostate histology WSIs.
+The implementation is based on:
+
+Andriy Myronenko, Ziyue Xu, Dong Yang, Holger Roth, Daguang Xu: "Accounting for Dependencies in Deep Learning Based Multiple Instance Learning for Whole Slide Imaging". In MICCAI (2021). [arXiv](https://arxiv.org/abs/2111.01556)
+
+![mil_patches](./mil_patches.jpg)
+![mil_network](./mil_network.jpg)
+
+## Requirements
+
+The script is tested with:
+
+- `Ubuntu 18.04` | `Python 3.6` | `CUDA 11.0` | `Pytorch 1.10`
+
+- the default pipeline requires about 16GB memory per gpu
+
+- it is tested on 4x16gb multi-gpu machine
+
+## Dependencies and installation
+
+### MONAI
+Please install the required dependencies
+```
+pip install tifffile
+pip install imagecodecs
+```
+
+
```

Comment:
There is an extra blank line here. Please remove it.

Reply:
ok